mkt/latecustomization/views.py

```diff
@@ -0,0 +1,50 @@
+from django.utils.datastructures import MultiValueDictKeyError
+
+from rest_framework import viewsets
+from rest_framework.exceptions import ParseError
+
+from mkt.api.authentication import (RestOAuthAuthentication,
+                                    RestSharedSecretAuthentication)
+from mkt.api.base import CORSMixin
+from mkt.constants.carriers import CARRIER_MAP
+from mkt.constants.regions import REGIONS_DICT
+from mkt.feed.permissions import FeedPermission
+from mkt.feed.views import FeedShelfPermissionMixin, RegionCarrierFilter
+from mkt.latecustomization.models import LateCustomizationItem
+from mkt.latecustomization.serializers import LateCustomizationSerializer
+
+
+class LateCustomizationViewSet(FeedShelfPermissionMixin, CORSMixin,
+                               viewsets.ModelViewSet):
```

Comment:
`ModelViewSet` has all the HTTP verbs, but you don't seem to want them ? (They are not documented / tested). From the docs it looks like you only need `GenericViewSet` and `mixins.DestroyModelMixin` ?